Resources/ServerInfo/Guidebook/Slang.xml

```diff
@@ -0,0 +1,35 @@
+<Document>
+  # Slang Terms
+  Space Station 14 players tend to use a lot of slang in their language to cut down on time spent typing.
+
+  ## Common IC (In Character) terms
+  - [color=#a4885c]Atmos[/color]: Shorthand for the atmospherics department.
+  - [color=#a4885c]AOS/KOS[/color]: Shorthand for Attack On Sight and Kill On Sight respectively. Both terms are commonly used by security.
+  - [color=#a4885c]Braindead[/color]: Used to describe someone who has not been present at their computer for a while (AFK). They may come back.
+  - [color=#a4885c]Cap[/color]:  Shorthand for Captain, the supreme authority of the station.
+  - [color=#a4885c]CentComm[/color]: Shorthand for Central Command, the agency which oversees station operations.
+  - [color=#a4885c]CE[/color]: Shorthand for Chief Engineer, head of the engineering and atmospherics departments.
+  - [color=#a4885c]CMO[/color]: Shorthand for Chief Medical Officer, head of the medical department.
+  - [color=#a4885c]Critical[/color]: Used to describe someone who has fallen unconscious due to damage. If any creature is in this state, they are likely either near death.
+  - [color=#a4885c]ERT[/color]: Shorthand for the Emergency Response Team, which central command may send to the station in the event of a crisis.
+  - [color=#a4885c]HoP[/color]: Shorthand for Head of Personnel, head of the service department and second in command under the captain.
+  - [color=#a4885c]HoS[/color]: Shorthand for Head of Security, head of the security department.
+  - [color=#a4885c]RD[/color]: Shorthand for Research Director, head of the R&D department.
+  - [color=#a4885c]QM[/color]: Shorthand for Quatermaster, head of the supply department.
```

Comment:
a LOT of these are very self explanatory. if you're going to do one for every head you might as well do one for every department. why is atmos included but not engi?